packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -302,20 +302,105 @@ List<Plugin> findPlugins(FlutterProject project) {
   }
   packages.forEach((String name, Uri uri) {
     final Uri packageRoot = uri.resolve('..');
-    final Plugin plugin = _pluginFromPubspec(name, packageRoot);
+    final Plugin plugin = _pluginFromPackage(name, packageRoot);
     if (plugin != null) {
       plugins.add(plugin);
     }
   });
   return plugins;
 }
 
+void _addPluginsToPlatform(List<Plugin> plugins, FlutterProjectPlatform platform, Map<String, dynamic> pluginsMap) {
+  if (!platform.existsSync()) {
+    return;
+  }
+  final List<Map<String,dynamic>> list = platform.pluginsList(plugins);
+  if (list.isNotEmpty) {
+    pluginsMap[platform.pluginConfigKey] = list;
+  }
+}
+
+/// Writes the .flutter-plugins.json file based on the list of plugins.
```

Comment:
`.flutter-plugins-dependencies` since new projects on stable have this file in the .gitignore already.

Reply:
I changed it to `.flutter-plugins-dependencies`, and won't break since I still add the `dependencyGraph` to support current projects. Once it's fully deprecated we can just remove it from the json